gulpfile.js

```diff
@@ -18,12 +18,23 @@ const exp = require('posthtml-expressions');
 const include = require('posthtml-include');
 
 gulp.task('clean', async () => {
-  return del([
-    'dist/**/*',
-  ]);
+  const dir = 'dist';
```

Comment:
Only using this once? Could either make use of it on line 28 or just get rid of it.

Reply:
Nice catch!